config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```diff
@@ -1845,3 +1845,47 @@ presubmits:
           limits:
             cpu: 4
             memory: 6Gi
+
+  - name: pull-kubernetes-node-e2e-containerd-dra
+    cluster: k8s-infra-prow-build
+    skip_branches:
+    - release-\d+\.\d+  # per-release image
+    always_run: false
+    run_if_changed: /dra/|/dynamicresources/|/resourceclaim/|/resourceclass/|/podscheduling/|/resourceclaimtemplate/|/dynamic-resource-allocation/|/pkg/apis/resource/|/api/resource/|/pkg/kubelet/cm/dra/|/kubelet/pkg/apis/dra/|/test/e2e_node/dra_test.go
```

Comment:
`/pkg/kubelet/cm/dra/|/kubelet/pkg/apis/dra/` is redundant, the initial `/dra/` already matches those directories.

Reply:
done